### PR TITLE
Abductor Clothes

### PR DESCRIPTION
**Pull request by Quantum-M**
:cl: 
add: Adds abductor jumpsuits for abductors.
/:cl:

Gives abductors a alien looking clothes instead of the lame, grey jumpsuit they have. The jumpsuits has some armor values in par with some of the science clothes.

The agent gets the purple suit, while the scientist gets the turquoise one.

EDIT: Vox cannot wear either, because I am not making the sprites for them.

abdcutorsuita
abductor suitb

bluejumpsuit

abductor team

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
I'm drunk but give the sci guy a turquoise suit and the agent a purple one or the other way around that will be dope thanks

**Comment by Ziiro** [comment]
>Vox cannot wear either, because I am not making the sprites for them.

YOUS IS SKREK

**Comment by Quantum-M** [comment]
@Ziiro 

kikikikikiki

Or however they say that

**Comment by Cactus2388** [comment]
They kinda look like space pimps

**Comment by AdamElTablawy** [comment]
no offense but they just look like space drag queens that fucked a TRON cosplay with this

File: code/game/gamemodes/miniantags/abduction/abduction_outfits.dm [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /datum/outfit/abductor
 	name = "Abductor Basic"
-	uniform = /obj/item/clothing/under/color/grey //they're greys gettit
```
Comment (FlattestGuitar) [inline code/game/gamemodes/miniantags/abduction/abduction_outfits.dm]: Make this `/obj/item/clothing/under/abductor` and remove the change from the agent.

Comment (Quantum-M) [reply code/game/gamemodes/miniantags/abduction/abduction_outfits.dm]: Done, @FlattestGuitar

**Comment by Quantum-M** [comment]
The abductor labcoats has been removed <s>and the files saved to a private area for potential future use</s>, you monsters.

Only the suits remain.

File: code/game/gamemodes/miniantags/abduction/abduction_gear.dm [hunk 2]
```diff
@@ -714,3 +714,23 @@ Congratulations! You are now trained for invasive xenobiology research!"}
...
+
+/obj/item/clothing/under/abductor/blue
+	desc = "The most advanced form of jumpsuit known to reality, looks uncomfortable. This one is in a turquoise color."
+	name = "alien jumpsuit"
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "abductor-suitblue"
+	item_state = "bl_suit"
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 10, bio = 10, rad = 0, fire = 0, acid = 0)
+	can_adjust = 0
+	species_restricted = list("exclude","Vox Outcast")
```
Comment (FlattestGuitar) [inline code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: everything that isn't changed on this jumpsuit is redundant. `.../abductor/blue` inherits everything from `.../abductor` unless explicitly overwritten

Comment (Quantum-M) [reply code/game/gamemodes/miniantags/abduction/abduction_gear.dm]: Done, @FlattestGuitar , removed the redundancies.

**Comment by Quantum-M** [comment]
I _think_  the conflicts should be fixed now.

**Comment by SamHPurp** [comment]
Travis appears to be having issues....?

**Pull request merged**